programs/witness_node/main.cpp

```diff
@@ -244,6 +245,13 @@ int main(int argc, char** argv) {
          create_new_config_file( config_ini_path, data_dir, cfg_options );
       load_config_file( config_ini_path, cfg_options, options );
 
+      if ( options.count("io-threads") )
```

Comment:
Please move this into application::initialize. (That has the added advantage that the option will also be interpreted correctly by the delayed_node executable.)

Reply:
Done